lab/lab1/part3-data-transformation.html

```diff
@@ -60,30 +60,29 @@
     Without modifying part4-data-dirty.js, change your code to account for the
     dirty data. Note: it is okay if your new code ignores a few of the points,
     if their formatting is especially wrong. But it should accommodate most points.
-
+;
     ===================== */
-
-    var data = bikeArrayClean;
-
+var data = bikeArrayClean;
+var data= bikeArrayDirty;
```

Comment:
You have not used and cleaned the dirty data set - that is is the challenging part about this lab..